cf-pubsub/src/main/java/org/eclipse/californium/pubsub/Converter.java

```diff
@@ -0,0 +1,264 @@
+/*******************************************************************************
+ * Copyright (c) 2018 RISE SICS and others.
```

Comment:
I'm not sure, if your work for the RISE SICS as well. 
If not, just place your university or employer.

Reply:
We've asked the Network Systems Lab and are awaiting their response on what to put under the copyright. I apologize for forgetting to change it.

Reply:
Any news on that?
If the "Network Systems Lab" doesn't claim the "copyright",  you may also just add the names of the authors. But for a clean legal state, It would be also great, if such a "no-claim" is written in some e-mail :-).

Reply:
The networks lab and KTH (same entity) both said they have no copyright. As for us, the authors, we've already signed our names. So I think it should be fine if the RISE copyright is removed and just our names left as authors.

Reply:
The eclipse foundation requires a explicit copyright statement.
It's not big thing, just list the same names above instead of the "RISE SICS".